common/learning/topic.ts

```diff
@@ -0,0 +1,27 @@
+import { match as Match, course_subject_enum as CourseSubject } from '@prisma/client';
+import { PrerequisiteError } from '../util/error';
+import { prisma } from '../prisma';
+import { User } from '../user';
+import { getLogger } from '../logger/logger';
+
+const logger = getLogger('LearningTopic');
+
+// Creates a new topic as a collection of notes and assignments
+export async function createTopic(user: User, match: Match, name: string, subject: CourseSubject) {
+    if (match.dissolved) {
+        throw new PrerequisiteError(`Match was already dissolved`);
+    }
+
+    const result = await prisma.learning_topic.create({
+        data: {
+            name,
+            subject,
```

Comment:
How did you get the error? The proper GraphQL query would be `subject: Deutsch_als_Zweitsprache` ... Think it is better to use the (flawed) course_subject enum here instead of introducing "yet another subject list" :)

Reply:
Oh nvm, the problem is not backend-related.
From the [frontend](https://github.com/corona-school/user-app/pull/615/files#diff-e22007e67e3afafa9627cfc817017864bb12d6e4b0e12173ec6a38810e013b09R42) we're using the name of the subject `Deutsch als Zweitsprache`.
I was testing with this UI
<img width="1440" alt="Bildschirmfoto 2024-09-10 um 12 33 50" src="https://github.com/user-attachments/assets/295e7367-8a67-462f-9a8f-9795b404dbc7">